code/api/src/modules/user/model.js

```diff
@@ -14,6 +14,18 @@ module.exports = function(sequelize, DataTypes) {
     },
     role: {
       type: DataTypes.TEXT
+    },
+    description: {
+      type: DataTypes.STRING
+    },
+    shippingAddress: {
+      type: DataTypes.DATEONLY
+    },
+    shipDate: {
+      type: DataTypes.DATEONLY
+    },
+    imageUrl: {
+      type: DataTypes.STRING
     }
   })
 
```

Comment:
Nice job.  Glad to know what this thing looks like now!  Now we should be able to make POST requests?

Reply:
You should be able to for email once Alex's user story is done. The other attributes wont work till we get through those user stories on our end.